client/state/themes/actions.js

```diff
@@ -404,6 +408,59 @@ export function clearActivated( siteId ) {
 }
 
 /**
+ * Triggers a network request to install WordPress.com theme on Jetpack site.
+ * After installataion it switches page to the customizer
+ * Requires Jetpack 4.4
+ *
+* @param  {String}   themeId      WP.com Theme ID
+ * @param  {String}   siteId       Jetpack Site ID
+ * @return {Function}              Action thunk
+ */
+function installAndTryAndCustomize( themeId, siteId ) {
+	return ( dispatch, getState ) => {
+		dispatch( {
+			type: THEME_TRY_AND_CUSTOMIZE_ON_JETPACK_REQUEST,
```

Comment:
I'd go as far as considering dropping state handling (i.e. dispatching `THEME_TRY_AND_CUSTOMIZE_ON_JETPACK_REQUEST*` in this function) and really just compose it from `installTheme` and that `page` call. I don't think we lose any information that way.